content/docs/reference-events.md

```diff
@@ -31,19 +31,19 @@ number timeStamp
 string type
 ```
 
-> Note:
+> Note :
 >
-> As of v0.14, returning `false` from an event handler will no longer stop event propagation. Instead, `e.stopPropagation()` or `e.preventDefault()` should be triggered manually, as appropriate.
+> À partir de la version 0.14, retourner `false` depuis un gestionnaire d'événements ne stoppe plus la propagation de l'événement. À la place, il convient de déclencher manuellement `e.stopPropagation()` ou `e.preventDefault()` selon le cas.
```

Comment:
```suggestion
> Depuis la version 0.14, renvoyer `false` depuis un gestionnaire d'événements n’interrompt plus la propagation de l'événement. Pour ce faire, appelez explicitement `e.stopPropagation()` ou `e.preventDefault()`, selon le besoin.
```

- “As of” est plus légèrement traduit par « Depuis »
- On ne retourne pas (notion spatiale de rotation), on renvoie (cf. Glossaire)
- « Il convient » n'a pas sa place si tout le monde n'est pas en costard noir cravate sombre 😉 
- « À la place » est pataud, et ici on peut tourner la phrase autrement plutôt que de chercher un synonyme moins ampoulé, ça donnera de meilleurs résultats.
- On allège considérablement la fin.